src/select.rs

```diff
@@ -700,12 +712,17 @@ mod tests {
         let expected_stmt = SelectStatement {
             tables: vec![Table::from("address")],
             fields: vec![
-                FieldExpression::Col(Column {
-                    name: String::from("max_addr"),
-                    alias: Some(String::from("max_addr")),
-                    table: None,
-                    function: Some(Box::new(agg_expr)),
-                }),
+                FieldExpression::new(
+                    Field::Col(Column {
+                        // TODO(ekmartin): name used to be alias here (max_addr).
+                        // Have to figure out if this is an okay breaking change,
+                        // or if we should somehow pass down the alias to Column.
+                        name: String::from("max(addr_id)"),
```

Comment:
@ms705: What do you think about this? Since the alias is now parsed by `FieldExpression`, `Column` never sees it, which means that we can't use it for naming (previously aliases would [override function names](https://github.com/ms705/nom-sql/pull/15/files#diff-20fbbdd72bf7e5c35269155c85c6ba39L518)).